src/Kunstmaan/AdminListBundle/Controller/AdminListController.php

```diff
@@ -456,6 +450,25 @@ private function getMaxSortableField($repo, $sort)
         return (int)$maxWeight;
     }
 
+    /**
+     * Sets the sort weight on a new item. Can be overridden if a non-default sorting implementation is being used.
+     *
+     * @param AbstractAdminListConfigurator $configurator The adminlist configurator
+     * @param $item
+     *
+     * @return mixed
+     */
+    protected function setSortWeightOnNewItem(AbstractAdminListConfigurator $configurator, $item) {
+        if ($configurator instanceof SortableInterface) {
```

Comment:
I moved the check for the SortableInterface here, since people using other sorting implementations shouldn't be required to implement this interface if their preferred sorting method does not require it.